servers/physics/collision_solver_sat.cpp

```diff
@@ -1632,6 +1632,51 @@ static void _collision_capsule_cylinder(const ShapeSW *p_a, const Transform &p_t
 
 	SeparatorAxisTest<CapsuleShapeSW, CylinderShapeSW, withMargin> separator(capsule_A, p_transform_a, cylinder_B, p_transform_b, p_collector, p_margin_a, p_margin_b);
 
+	if (!separator.test_previous_axis()) {
+		return;
+	}
+
+	// Cylinder B end caps.
+	Vector3 cylinder_B_axis = p_transform_b.basis.get_axis(1).normalized();
+	if (!separator.test_axis(cylinder_B_axis)) {
+		return;
+	}
+
+	// Cylinder edge against capsule balls.
+
+	Vector3 capsule_A_axis = p_transform_a.basis.get_axis(2);
```

Comment:
You're using `get_axis(1)` in the `master` PR, and `get_axis(2)` here. Is that expected?

Reply:
Comparing more of the two files it seems to be a common pattern so I guess it's on purpose.

Reply:
Yeah it's on purpose. It's because capsules have been changed to vertical direction on master.